microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -3,102 +3,212 @@ description: The latest experimental features in Microsoft Edge DevTools
 keywords: microsoft edge, web development, f12 tools, devtools, experiment
 no-loc: ["Enable webhint", "Enable Network Console", "Source Order Viewer", "Enable Composited Layers in 3D View", "Enable new Font Editor tool within the Styles pane", "Enable new CSS Flexbox debugging features", "Enable + button tab menus to open more tools", "Enable Welcome tab", "3D View", "Turn on support to move tabs between panels", "Match keyboard shortcuts in the DevTools to Microsoft Visual Studio Code", "Edit keyboard shortcuts for any action in the DevTools", "Turn on new CSS grid debugging features", "Emulation: Support dual screen mode"]
 ---
 # Experimental features
 
-Microsoft Edge DevTools provide access to experimental features that are still in development.  You can try out these features and [provide feedback](../contact.md) before each feature is released.
+Microsoft Edge DevTools provide access to experimental features that are still in development. This article describes only experimental features found in Microsoft Edge. You can try out these features and [provide feedback](../contact.md) before each feature is released.
 
-All channels of Microsoft Edge have experimental features.  You can get the latest experimental features by using the Microsoft Edge Canary channel.  This article only covers selected experimental features.  For the full list, see the **Settings** > **Experiments** page in DevTools.
+All [channels of Microsoft Edge](/deployedge/microsoft-edge-channels) have experimental features. You can get the latest experimental features by using the [Microsoft Edge Canary channel](https://www.microsoftedgeinsider.com/welcome?channel=canary). To view the full list available in your version of Microsoft Edge, see the **Settings** > **Experiments** page in DevTools.
 
 
 <!-- ====================================================================== -->
-## Experimental features which are turned on by default
-
-As of Microsoft Edge 94, the following experimental features are turned on by default.  This means that these features can be used right away, without you having to change any settings.  You can turn off these default experimental features, if needed.
-
-<!-- listed in this list in order per Settings > Experiments pane -->
-*  Source order viewer
-*  [Emulation: Support dual screen mode](../device-mode/dual-screen-and-foldables.md) is turned on by default starting with Microsoft Edge 90.
-*  Enable webhint
-*  Show issues in Elements
-*  Enable Composited Layers in 3D View
-*  DevTools Tooltips
-*  Visual Studio Code themes for the DevTools
-*  Enable keyboard shortcut editor - [Edit keyboard shortcuts for any action in the DevTools](../customize/shortcuts.md#edit-the-keyboard-shortcut-for-a-devtools-action) is turned on by default starting with Microsoft Edge 89.
-
+## Experimental features in the current release
+
+The following table lists the default status of each experimental feature. The feature name links to more information.
+
```

Comment:
Maybe eliminate the table and place the default state checkbox info in each h2 section below.  The table is manually duplicating the "In this article" navigation feature, difficult to maintain.

Reply:
fixed

Reply:
deleted table